frontend/src/LogOutButton/LogOutButton.tsx

```diff
@@ -0,0 +1,13 @@
+import useAuth from "../Hooks/useAuth";
+import axios from "axios";
+
+export default function LogOutButton(){
+    const user = useAuth(false)
+    function handleLogOut() {
+        axios.post('/api/users/logout')
+            .then(() => window.location.href = '/login')
```

Comment:
use useNavigate() here

Reply:
Tried it but couldn't get the page to refresh despite rewatching the logout video and adding a layout component :-/